src/org/opendatakit/briefcase/operations/Export.java

```diff
@@ -136,7 +136,7 @@ public static void export(String storageDir, String formid, Path exportDir, Stri
             ).map(Credentials::from)
         );
 
-        pullJob = PullForm.pull(CommonsHttp.reusing(), server, briefcaseDir, false, Export::onEvent, formStatus);
+        pullJob = PullForm.pull(CommonsHttp.of(8), server, briefcaseDir, false, Export::onEvent, formStatus);
```

Comment:
I know the literal 8 is temporary, but it’s explained only in a commit comment (that I can see). Perhaps it should be extracted from where it appears into a constant, whose name expresses its temporary nature, or marked with a // todo. (I don’t feel strongly about this.)

Reply:
I think you're right. I'll move it to a constant in `Launcher` (topmost class)